static/_redirects

```diff
@@ -1,675 +1,245 @@
+/docs/integrations/gitlab.html /docs/integrations/user-identity/gitlab
+/docs/integrations/google /docs/integrations/user-identity/google
+/docs/integrations/google.html /docs/integrations/user-identity/google
+/docs/integrations/index /docs/integrations/user-identity/identity-providers
+/docs/integrations/index.html /docs/integrations/user-identity/identity-providers
+/docs/integrations/okta /docs/integrations/user-identity/okta
+/docs/integrations/okta.html /docs/integrations/user-identity/okta
+/docs/integrations/one-login /docs/integrations/user-identity/one-login
+/docs/integrations/one-login.html /docs/integrations/user-identity/one-login
+/docs/integrations/ping /docs/integrations/user-identity/ping
+/docs/integrations/ping.html /docs/integrations/user-identity/ping
+/docs/integrations/bamboohr /docs/integrations/user-standing/bamboohr
+/docs/integrations/bamboohr.html /docs/integrations/user-standing/bamboohr
+/docs/capabilities/directory-sync /docs/integrations/user-standing/directory-sync
+/docs/capabilities/directory-sync.html /docs/integrations/user-standing/directory-sync
+/docs/integrations/zenefits /docs/integrations/user-standing/zenefits
+/docs/integrations/zenefits.html /docs/integrations/user-standing/zenefits
+/docs/capabilities/certificates-and-tls /docs/internals/certificates-and-tls
+/docs/capabilities/certificates-and-tls.html /docs/internals/certificates-and-tls
+/docs/manage/clusters /docs/internals/clusters
+/docs/manage/clusters.html /docs/internals/clusters
+/docs/capabilities/high-availability /docs/internals/configuration
+/docs/capabilities/high-availability.html /docs/internals/configuration
+/docs/capabilities/enterprise-api /docs/internals/management-api-enterprise
+/docs/capabilities/enterprise-api.html /docs/internals/management-api-enterprise
+/docs/deploy/cloud/api-guide /docs/internals/management-api-zero
+/docs/deploy/cloud/api-guide.html /docs/internals/management-api-zero
+/docs/manage/mutual-auth /docs/internals/mutual-auth
+/docs/manage/mutual-auth.html /docs/internals/mutual-auth
+/docs/capabilities/metrics /docs/internals/metrics
+/docs/capabilities/metrics.html /docs/internals/metrics
+/docs/capabilities/ppl /docs/internals/ppl
+/docs/capabilities/ppl.html /docs/internals/ppl
+/docs/capabilities/programmatic-access /docs/internals/programmatic-access
+/docs/capabilities/programmatic-access.html /docs/internals/programmatic-access
+/docs/manage/troubleshooting /docs/internals/troubleshooting
+/docs/manage/troubleshooting.html /docs/internals/troubleshooting
+
+# Additional old capabilities => new locations
+/docs/capabilities/rego /docs/internals/ppl
+/docs/capabilities/namespacing /docs/internals/namespacing
+/docs/capabilities/load-balancing /docs/capabilities/routing
+/docs/capabilities/device-identity /docs/integrations/device-context/device-identity
+/docs/capabilities/single-sign-out /docs/capabilities/authentication
+/docs/capabilities/mtls-clients /docs/internals/mutual-auth
```

Comment:
We do still have https://docs.pomerium.com/docs/reference/downstream-mtls-settings if we want to link to that. Seems like we got rid of the upstream mTLS page, but kept the downstream one.

Reply:
Merged to:
/docs/capabilities/mtls-clients /docs/internals/certificates-and-tls#downstream-tls
/docs/capabilities/mtls-services /docs/internals/certificates-and-tls#upstream-tls